.travis.yml

```diff
@@ -1,9 +1,22 @@
-language: node_js
-node_js:
-  - "6"
+language: java
+jdk:
+- oraclejdk8
+before_install:
+- java -version
+- wget https://raw.githubusercontent.com/creationix/nvm/v0.31.0/nvm.sh -O ~/.nvm/nvm.sh
+- source ~/.nvm/nvm.sh
+- nvm install 6.9.1
+- node --version
+- npm install
 services:
 - postgresql
 addons:
   postgresql: "9.4"
+script:
+- source ./scripts/OGC_tests/install_TE.sh
+- source ./scripts/OGC_tests/configure_TE_BASE.sh
+- source ./scripts/OGC_tests/configure_sta10_tests.sh
+- source ./scripts/OGC_tests/launch_tests.sh
```

Comment:
Yes, I know, I am trying without running the server as a test, the same in local just to check that the tests start as they should, giving the connection error message. But instead the error is different, there is something wrong when launching them not related with the server being launched.